mesonbuild/cmake/traceparser.py

```diff
@@ -479,7 +497,10 @@ def _lex_trace(self, trace):
             func = mo_file_line.group(4)
             args = mo_file_line.group(5)
             args = parse_generator_expressions(args)
-            args = args.split(' ')
+            try:
+                args = shlex.split(args)
+            except:
```

Comment:
__[Flake8]__ [B001] Do not use bare `except:`, it also catches unexpected events like memory errors, interrupts, system exit, and so on.  Prefer `except Exception:`.  If you're sure what you're doing, be explicit and write `except BaseException:`. ([link](https://sider.review/gh/repos/19784232/pulls/5989#issue-4637199))

<sub>posted by Sider</sub>